docs/guide/Core_Component.md

```diff
@@ -13,6 +13,13 @@ _Includes: Ballot Encryption_
 
 :fontawesome-brands-github: Source - ✨ Coming Soon
 
+## Java
+
+_Includes: Complete port of python library_
```

Comment:
You can include this line
`Includes: Key Generation, Ballot Encryption, Tallying, Decryption`

Reply:
done

Reply:
Whats with ":fontawesome-brands-github:"? It doesnt seem to resolve when I look at the page on github??

Reply:
We use github pages with a markdown theme. 

https://www.electionguard.vote/

Reply:
https://www.electionguard.vote/guide/Core_Component/